### PR TITLE
Improves plugin list filters

### LINKED ISSUE #11145: Plugin Filter "third party" shows several plugins with author "piwik"
Plugin Filter "third party" shows several plugins with author "piwik"
that's somehow irritating....
(piwikPro should be fine)
please see attachment

2017-01-04_13h17_34

### PR DESCRIPTION
**Pull request by sgiehl**
As the available filters  (All, Core, Third-Party) were kind of confusing, I tried to improve that by adding a new category 'Official', which will list all plugins that are developed by the Piwik team

image

Name of this category is still open for discussion.
Also we might consider to add a new category 'Premium', listing all plugins bought on Marketplace. Any opinions on that?

fixes #11145

### CODE REVIEW

File: plugins/CorePluginsAdmin/Controller.php [hunk 1]
```diff
@@ -219,6 +219,16 @@ protected function getPluginsInfo($themesOnly = false)
         foreach ($plugins as $pluginName => &$plugin) {
 
             $plugin['isCorePlugin'] = $this->pluginManager->isPluginBundledWithCore($pluginName);
+            $plugin['isOfficialPlugin'] = false;
+
+            if (isset($plugin['info']) && isset($plugin['info']['authors'])) {
+                foreach ($plugin['info']['authors'] as $author) {
+                    if ($author['name'] == 'Piwik' || $author['name'] == 'InnoCraft') {
```
Comment (tsteur) [inline plugins/CorePluginsAdmin/Controller.php]: maybe good to check lowercase here as it may change in the future (shouldn't but never know)

**Comment by tsteur** [comment]
Sounds good 👍  Either official or `First-party` could maybe work as well as it says next to it Third-party.

File: plugins/CorePluginsAdmin/Controller.php (hunk 1 above)
Comment (sgiehl) [inline plugins/CorePluginsAdmin/Controller.php]: changed.

**Comment by mattab** [comment]
Excellent :+1:

**Pull request merged**